<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 48, 98);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lwEmployees">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>50</y>
      <width>256</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Double click an employee to delete them&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
border: 1px solid white;

</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pBaddEmployee">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:rgb(255, 255, 255);
border: 1px solid white;
}

QPushButton:hover {
background-color: rgb(67, 85, 133);
}</string>
    </property>
    <property name="text">
     <string>Add Employee</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>143</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(67, 85, 133);
color:rgb(255, 255, 255);
border-left: 1px solid white;
border-top: 1px solid white;
border-bottom: 1px solid white;
padding-top: 17px;</string>
    </property>
    <property name="text">
     <string>Tuesday</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(67, 85, 133);
color:rgb(255, 255, 255);
border-top: 1px solid white;
border-bottom: 1px solid white;
padding-top: 17px;</string>
    </property>
    <property name="text">
     <string>Wednesday</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>143</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(67, 85, 133);
color:rgb(255, 255, 255);
border-top: 1px solid white;
border-bottom: 1px solid white;
padding-top: 17px;</string>
    </property>
    <property name="text">
     <string>Thursday</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>143</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(67, 85, 133);
color:rgb(255, 255, 255);
border-top: 1px solid white;
border-bottom: 1px solid white;
padding-top: 17px;</string>
    </property>
    <property name="text">
     <string>Friday</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>143</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(67, 85, 133);
color:red;
border-top: 1px solid white;
border-bottom: 1px solid white;
padding-top: 17px;</string>
    </property>
    <property name="text">
     <string>Saturday</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>150</y>
      <width>143</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(67, 85, 133);
color:red;
border-top: 1px solid white;
border-bottom: 1px solid white;
padding-top: 17px;</string>
    </property>
    <property name="text">
     <string>Sunday</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>150</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(67, 85, 133);
color:white;
border-top: 1px solid white;
border-right: 1px solid white;
border-bottom: 1px solid white;
padding-top: 17px;</string>
    </property>
    <property name="text">
     <string>Monday</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(67, 85, 133);
color:rgb(255, 255, 255);
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>Employees</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumEmployees">
    <property name="geometry">
     <rect>
      <x>1225</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(67, 85, 133);
color:rgb(255, 255, 255);
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pBcalculateShiftsForMonth">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:rgb(255, 255, 255);
border: 1px solid white;
}

QPushButton:hover {
background-color: rgb(67, 85, 133);
}</string>
    </property>
    <property name="text">
     <string>Calculate Shift Lists</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBtuesday">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 1px solid white;
focus:none;
selection-background-color: rgb(54, 48, 98);

</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBwednesday">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 1px solid white;
focus:none;
selection-background-color: rgb(54, 48, 98);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBthursday">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 1px solid white;
focus:none;
selection-background-color: rgb(54, 48, 98);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBfriday">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 1px solid white;
focus:none;
selection-background-color: rgb(54, 48, 98);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBsaturday">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 1px solid white;
focus:none;
selection-background-color: rgb(54, 48, 98);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBsunday">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>150</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 1px solid white;
focus:none;
selection-background-color: rgb(54, 48, 98);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBmonday">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>150</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 1px solid white;
focus:none;
selection-background-color: rgb(54, 48, 98);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBnumDays">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 1px solid white;
focus:none;
selection-background-color: rgb(54, 48, 98);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>365</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(54, 48, 98);
color:rgb(255, 255, 255);
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter The Number of Days &lt;br/&gt;Below to Calculate Shift Lists&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>1001</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(54, 48, 98);
color:rgb(255, 255, 255);
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>Change The SpinBox Values Below To Indicate How Many Employees Should Work On Each Day</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tWshiftLists">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>1001</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QAbstractItemView{
color:white;
border:1px solid white;
selection-background-color:rgb(54, 48, 98);
}

QHeaderView::section{
background-color:rgb(54, 48, 98);
color:white;
}</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
